xbmc/utils/StreamDetails.cpp

```diff
@@ -557,8 +557,8 @@ std::string CStreamDetails::VideoDimsToResolutionDescription(int iWidth, int iHe
   // 1280x720
   else if (iWidth <= 1280 && iHeight <= 720)
     return "720";
-  // 1920x1080
-  else if (iWidth <= 1920 && iHeight <= 1080)
+  // 1920x1080 (sometimes 1088, which is a multiple of 16 - see comment about 544)
```

Comment:
Note that 1088 would be scaled to 1080 which ruins quality. So if there is really 1088 material out there, it rather deserves a label "crap" than 1080